images/win/scripts/Installers/Install-AWS.ps1

```diff
@@ -13,6 +14,14 @@ Install-Binary -Url $sessionManagerUrl -Name $sessionManagerName -ArgumentList (
 $env:Path = $env:Path + ";$env:ProgramFiles\Amazon\SessionManagerPlugin\bin"
 
 # Install AWS SAM CLI
-Install-Binary -Url "https://github.com/awslabs/aws-sam-cli/releases/latest/download/AWS_SAM_CLI_64_PY3.msi" -Name "AWS_SAM_CLI_64_PY3.msi"
+$packageName = "AWS_SAM_CLI_64_PY3.msi"
+$repoURL = "https://github.com/awslabs/aws-sam-cli/releases/latest"
+$packagePath = Start-DownloadWithRetry -Url "$repoURL/download/$packageName" -Name $packageName
```

Comment:
`repoURL` variable is used only once to create URL for download. Why not defining download URL instead and avoid extra string expansion?

```suggestion
$packageUrl = "https://github.com/awslabs/aws-sam-cli/releases/latest/download/$packageName"
$packagePath = Start-DownloadWithRetry -Url $packageUrl -Name $packageName
```